gen_schema.py

```diff
@@ -55,7 +57,26 @@ def field_schema(field: ModelField, **kwargs: Any) -> Any:
     by_alias=True,
     indent=4,
 ).replace("schema_type", "type")
-write(schema, Path("schema_pipeline.json"))
+path_schema_pipeline = Path("schema_pipeline.json")
+write(schema, path_schema_pipeline)
 
 schema = schema_json_of(PipelineConfig, title="kpops config schema", indent=4)
-write(schema, Path("schema_config.json"))
+path_schema_config = Path("schema_config.json")
+write(schema, path_schema_config)
+
+# Generate json file for editor integration
```

Comment:
good idea about generating the `settings.json`, but as long as the url doesn't change this is actually unnecessary.

Reply:
Yeah, I admit it is a bit of an overkill. The main goal of this was to avoid having to edit the doc page at all, but I ended up not achieving that haha. I thought of also generating the list of supported files, but the place likely isn't `gen_schema.py` and the list of all possibly supported files isn't long either.

Reply:
done